.gitpod.yml

```diff
@@ -0,0 +1,9 @@
+# List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
+tasks:
```

Comment:
<img width="1638" alt="Screen Shot 2021-11-11 at 11 39 48 AM" src="https://user-images.githubusercontent.com/19964402/141358617-11ec8088-1423-465d-af94-7803a938d060.png">

This is what working with it looks/feels like :)